docs/cookbook/adding-a-new-advert-position.md

```diff
@@ -0,0 +1,51 @@
+# Adding a New Advert Position
+
+By default, you can place advert banner on predefined positions:
+- under the page header
+- in the left panel (under the category tree)
+- above the page footer
+- in a category detail (above the category name)
+
+All the positions are defined in `AdvertPositionRegistry` class 
+that provides them for displaying on adverts list in admin
+as well as for options for selectbox on advert detail administration page.
+
+![Advert detail in administration](img/new-advert-screen.png)
+
+If you want to add a new custom position, let us say on product detail page, you need to follow these steps:
+
+1. Create a new class that extends `AdvertPositionRegistry` and override `getAllLabelsIndexedByNames()` method where you add a translated description for your new advert position to the existing ones.
+    ```php
+    <?php
+    
+    namespace Shopsys\ShopBundle\Model\Advert;
+    
+    use Shopsys\FrameworkBundle\Model\Advert\AdvertPositionRegistry as BaseAdvertPositionRegistry;
+    
+    class AdvertPositionRegistry extends BaseAdvertPositionRegistry
+    {
+        /**
+         * @return string[]
+         */
+        public function getAllLabelsIndexedByNames(): array
+        {
+            $advertPositions = parent::getAllLabelsIndexedByNames();
+            $advertPositions['productDetail'] = t('on product detail page');
+            
+            return $advertPositions;
+        }
+    }
+     ```
+
+
+2. In your `services.yml`, set your new class as an alias for the base one.
+    ```yaml
+    Shopsys\ShopBundle\Model\Advert\AdvertPositionRegistry: ~
+    
+    Shopsys\FrameworkBundle\Model\Advert\AdvertPositionRegistry: '@Shopsys\ShopBundle\Model\Advert\AdvertPositionRegistry'
+    ```
+
+3. Render the new advert on the product detail page (or wherever else you want) by adding following line into the corresponding twig template.
+    ```twig
+    {{ render(controller('ShopsysShopBundle:Front/Advert:box',{'positionName' : 'productDetail'})) }}
+    ```
```

Comment:
Some kind of conclusion (maybe with an illustration screenshot?) is missing at the end.

Otherwise, the cookbook is perfect! :medal_sports: